test/unit/math/src/sincos.cpp

```diff
@@ -7,7 +7,7 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
```

Comment:
The copyright header needs an update here.

Reply:
done